createdisk.sh

```diff
@@ -61,6 +61,13 @@ function create_qemu_image {
     rm -f $destDir/${CRC_VM_NAME}.qcow2
     mv $destDir/${CRC_VM_NAME}_sparse.qcow2 $destDir/${CRC_VM_NAME}.qcow2
 
+    # Before using the created qcow2 check if it have lazy_refcounts true.
+    lazy_refcounts=$(${QEMU_IMG} info ${destDir}/${CRC_VM_NAME}.qcow2 | grep -oP "lazy refcounts: true")
+    if [ -z $lazy_refcounts ]; then
+        echo "${CRC_VM_NAME}.qcow2 doesn't have lazy_refcounts enabled"
```

Comment:
Maybe also explain in the error message that "This is going to cause disk image corruption when using hyperkit"?